trade-finance-manager-ui/server/controllers/case/parties/index.test.js

```diff
@@ -353,16 +354,113 @@ describe('controllers - case - parties', () => {
             _id: mockDeal._id,
           },
           body: {
-            exporter: {
-              partyUrn: '12345',
-            },
+            partyUrn: '12345',
```

Comment:
Should this be not inside `exporter` like other parties?

Reply:
discussed.  Sits outside in the body just as partyUrn